tests/ert_tests/ert3/data/test_record.py

```diff
@@ -102,11 +138,14 @@ def test_inconsistent_size_ensemble_record(raw_ensrec, ensemble_size):
     ("ensemble_size", "raw_multiensrec"),
     (
         (
-            3,
+            4,
             {
                 "ens1": {"records": [{"data": [1, 2, 3]} for _ in range(3)]},
                 "ens2": {"records": [{"data": {"a": 1, "b": 2}} for _ in range(3)]},
                 "ens3": {"records": [{"data": [0]}, {"data": [2]}, {"data": [1]}]},
+                "ens4": {
+                    "records": [{"data": [b""]}, {"data": [b"1"]}, {"data": [b"abc"]}]
+                },
```

Comment:
Now with the slightly stronger type definition of the ensemble record, in that it should be either only blob or numerical records, could we have a test verifying that a list with at least one blob record and at least one numerical record actually causes an error? That is behaviour we probably want to preserve even if the implementation itself changes...

Reply:
done.